.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
     steps:
```

Comment:
I think you specify working directory like this:
```
working_directory: ~/pisp_app
steps:
  - checkout
```

Reply:
I missed this comment. I'll fix it in the other PR after rebasing.

Reply:
I also think we should change the root of the project to be pisp_app itself. Any thoughts ?